docs/content/manual/upgrade/upgrade-from-new-network-design.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Upgrade Harvester from new cluster network design (after v1.1.0)
+---
+
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
+1. Network have at least two NICs
+1. Suggest each Harvester node have at lease 6 core CPU, 16GB memory and 200GB disk
```

Comment:
Same as above comment, have the minimum required hardware.

Reply:
Remove the line of minimum required hardware